core/src/main/scripts/import_data_validator/validateData.py

```diff
@@ -1253,6 +1405,38 @@ def checkLine(self, data):
                                'cause': value})
                 self.sampleIds.add(value.strip())
 
+    @classmethod
+    def request_attrs(cls, logger):
+        '''Set cls.srv_sample_attrs and cls.srv_patient_attrs from portal.'''
+        if cls.srv_attrs is None:
+            cls.srv_attrs = cls._request_attrs(
+                SERVER_URL + '/api/clinicalattributes/patients', logger)
+            srv_sample_attrs = cls._request_attrs(
+                SERVER_URL + '/api/clinicalattributes/samples', logger)
+            id_overlap = (set(cls.srv_attrs.keys()) &
+                          set(srv_sample_attrs.keys()))
+            if id_overlap:
+                raise ValueError(
```

Comment:
should not be an error here. If we want such a constraint, we should add it to the DB itself.